source/analytics/superset/react-with-superset/backend-node/backend-node.js

```diff
@@ -0,0 +1,78 @@
+const express = require('express')
+const fetch = (...args) => import('node-fetch').then(({default:fetch})=> fetch(...args));
+
+
+const PORT = 3001
+const app = express()
+
+app.listen(PORT, () => {
+  console.log(`Server running on port ${PORT}`)
+})
+
+async function fetchAccessToken() {
+  try {
+    const body = {
+      username: "username-sample", // admin username
+      password: "password-sample", // admin password
+      provider: "db",
+      refresh: true,
+    }
+
+    const response = await fetch(
+      "http://localhost:8088/api/v1/security/login",
```

Comment:
Usually this will be in a config file right

Reply:
> Usually this will be in a config file right

We need to validate this on an external server by providing the IP or DNS name. localhost will not work in real world scenario.

Reply:
> Usually this will be in a config file right

That's a good point. I have not set up a config file for this project but I can update it. Would it be better to do it in this PR or future PR?

Reply:
Sounds good. Thanks for the input!